Example/Tests/EthereumRequestTests.swift

```diff
@@ -69,31 +70,52 @@ class EthereumRequestTests: XCTestCase {
         XCTAssertEqual(socketRep["parameters"] as? [String], params)
     }
     
-    func testStructSocketRepresentation() {
```

Comment:
Add new test cases but not delete the existing ones. This one is testing when using a custom struct to make request i.e `Transaction`.